### PR TITLE
Ignore visibleForTesting lint

### PR DESCRIPTION
**Pull request by tvolkert**

### CODE REVIEW

**Review by matthew-carroll — APPROVED** [review]

File: packages/flutter_driver/lib/src/common/fuchsia_compat.dart [hunk 1]
```diff
@@ -97,6 +97,6 @@ class FuchsiaCompat {
   static Future<FuchsiaRemoteConnection> connect() async {
     FuchsiaCompat._init();
     return FuchsiaRemoteConnection
-        .connectWithSshCommandRunner(_DummySshCommandRunner());
+        .connectWithSshCommandRunner(_DummySshCommandRunner()); // ignore: invalid_use_of_visible_for_testing_member
```
Comment (awdavies) [inline packages/flutter_driver/lib/src/common/fuchsia_compat.dart]: I thought the visibleForTesting/meta etc was removed from FRDP?

Comment (tvolkert) [reply packages/flutter_driver/lib/src/common/fuchsia_compat.dart]: Ah, maybe it's just that it needs to be rolled into google.  I'll look into that first.

**Comment by Hixie** [comment]
If we need to access the member for non-testing purposes, then this indicates that it shouldn't be marked as `@visibleForTesting`, surely?

**Comment by tvolkert** [comment]
Yeah the annotation had since been removed - we just needed to update google's repo with the latest code.

**Pull request closed without merging**